do_snowDA.sh

```diff
@@ -181,16 +224,27 @@ fi
 
 if [[ $do_DA == "YES" ]]; then 
 
-    cp -r ${RSTRDIR} $WORKDIR/mem_pos
-    cp -r ${RSTRDIR} $WORKDIR/mem_neg
+    for (( ensemble_count=0; ensemble_count<ensemble_size; ensemble_count++ ))
+    do
+        if [ $do_ens == "YES" ]; then
+            ens_member=${ens_list[$ensemble_count]}
+            ENSEMBLE_DIR=${WORKDIR}/${ens_member}
+        else
+            ENSEMBLE_DIR=${WORKDIR}
+        fi
+        RSTRDIR=$ENSEMBLE_DIR/restarts/tile
+
+        ${CP} -r ${RSTRDIR} $ENSEMBLE_DIR
+        echo "Trace 06 restart file in "${SCRIPT}": everything in ${ENSEMBLE_DIR} are copied from "${RSTRDIR}
+    done
 
     echo 'snowDA: calling create ensemble' 
 
-    python ${SCRIPTDIR}/letkf_create_ens.py $FILEDATE $B
-    if [[ $? != 0 ]]; then
-        echo "letkf create failed"
-        exit 10
-    fi
+#   python ${SCRIPTDIR}/letkf_create_ens.py $FILEDATE $B
```

Comment:
This should not be commented out. Instead, enclose it in a loop. If do_DA = "LETKF-OI" we do this.